src/arr/locker/~tables.arr

```diff
@@ -0,0 +1,86 @@
+#lang pyret/library
+
```

Comment:
This PR is created last year. A lot of things have changed and when we came back to it recently we had a bad merge. So, yes, the file you mentioned is not intended to be included in this PR. It must be cleaned before merging.